### ISSUE TITLE
[Request] Tile blocks adaptive textures

### ISSUE THREAD

**Issue by raymondbh** [Chisel-Team/CTMLib#3]
Is it possible to make the CTMLib put together a block texture from different textures depending on how the block is crafted?
Say I have 16 items of different tiles I would like to combine in any combination, and the block would take the texture from the different items it was put together with....? Not sure if you understand what I mean, English is not my first languages, I will try to explain better if you don't...

**Comment by tterrag1098** [comment]
This could be done with a custom submap manager that returns icons based on the item.

**Comment by raymondbh** [comment]
Is this something that could find its way into CTMLib? :P

**Comment by Drullkus** [comment]
I doubt it. You'd have to write your own submap manager.

**Comment by tterrag1098** [comment]
No, because there's no way it could be done generically. Just write it yourself.
